sandbox-seed.json

```diff
@@ -1,109 +1,130 @@
 {
   "circulars": [
     {
+      "dummy": 0,
```

Comment:
I fixed this on the main branch. Please rebase.

Reply:
Sweet, rebased